### ISSUE TITLE
infinite ajax scroll + salvattore

### ISSUE THREAD

**Issue by starise** [webcreate/infinite-ajax-scroll#178]
Thanks for this nice piece of code. I use [salvattore](http://salvattore.com) to arrange my items in columns. Now i'm trying to use it in combination with IAS. I'm calling salvattore on 'render' event, and this works, but only for the first "next" page of elements, then it stops loading.

``` javascript

 <script type="text/javascript">
   var ias = $.ias({
     container: ".boxes",
     item: ".box",
     pagination: "nav.navigation",
     next: ".nav-next a",
   });
  ias.on('render', function(items) {
    var grid = $('.boxes')[0];
    salvattore.append_elements(grid, items);
    // ias.fire('rendered', [items]);
    return false;
  })
```

If i return true instead of false on 'render', it works, but salvattore doesn't arranges the items and i have all of these appended on the second column (2 column layout).

Any idea?

**Comment by fieg** [comment]
The `rendered` event not being fired when you return false from the `render` event might have something to do with it. I see you commented it out. It didn't work?

**Comment by fieg** [comment]
A potential fix has been merged: #199
